docs/source/json-api/index.rst

```diff
@@ -819,15 +826,47 @@ HTTP Response
 .. code-block:: json
 
     {
-        "status": 200,
-        "result": [
-            {
-                "party": "Alice",
-                "isLocal": true
-            }
-        ]
+      "status": 200,
+      "result": [
+        {
+          "identifier": "Alice",
+          "displayName": "Alice & Co. LLC",
+          "isLocal": true
+        },
+        {
+          "identifier": "Bob",
+          "displayName": "Bob & Co. LLC",
+          "isLocal": true
+        },
+        {
+          "identifier": "Dave",
+          "isLocal": true
+        }
+      ]
     }
 
+Please note that the order of the party objects in the response is not guaranteed to match the order of the passed party identifiers.
+
+An empty JSON array will be returned in the ``result`` element of the response if passed identifiers did not match any party.
```

Comment:
If we clarify what happens with unknown parties above, this should not be needed anymore.